server/memphis_handlers_consumers.go

```diff
@@ -292,8 +281,8 @@ func (s *Server) createConsumerDirect(c *client, reply string, msg []byte) {
 		return
 	}
 
-	err = s.createConsumerDirectCommon(c, ccr.Name, ccr.StationName, ccr.ConsumerGroup, ccr.ConsumerType, ccr.ConnectionId, ccr.MaxAckTimeMillis, ccr.MaxMsgDeliveries, 1, ccr.StartConsumeFromSequence, ccr.LastMessages)
-	respondWithErr(serv.MemphisGlobalAccountString(), s, reply, err)
+	err = s.createConsumerDirectCommon(c, ccr.Name, ccr.StationName, ccr.ConsumerGroup, ccr.ConsumerType, ccr.ConnectionId, tenantName, ccr.Username, ccr.MaxAckTimeMillis, ccr.MaxMsgDeliveries, 1, ccr.StartConsumeFromSequence, ccr.LastMessages)
+	respondWithErr(MEMPHIS_GLOBAL_ACCOUNT, s, reply, err)
```

Comment:
change MEMPHIS_GLOBAL_ACCOUNT to serv.MemphisGlobalAccountString() in all relevant places